pure.zsh

```diff
@@ -271,7 +271,10 @@ prompt_pure_async_vcs_info() {
 	# and stash information via misc (%m).
 	zstyle ':vcs_info:git*' formats '%b' '%R' '%a' '%m'
 	zstyle ':vcs_info:git*' actionformats '%b' '%R' '%a' '%m'
-	zstyle ':vcs_info:git*+set-message:*' hooks git-stash
+	zstyle -t ":prompt:pure:git:stash" show
```

Comment:
One thing to keep in mind is that since this is running inside the async worker. Retroactive changes to zstyle won’t propagate to here. A better option might be to pass a boolean argument (1/0) to this function when it’s invoked (as async job)

Reply:
Would that be in that line:

```
async_job "prompt_pure" prompt_pure_async_vcs_info
```

?

Reply:
That’s right. Read the value before and then pass it along there :).

Reply:
Fixed..